lib/streamlit/elements/plotly_chart.py

```diff
@@ -290,8 +289,7 @@ def marshall(
     ctx = get_script_run_ctx()
     id = compute_widget_id(
         "plotly_chart",
-        user_key=key,
-        figure_or_data=figure_or_data,
+        figure_or_data=figure_or_data,  # type: ignore
```

Comment:
Why do we need an type ignore here?

Reply:
Added the types to compute_widget_id's typing function.